.github/workflows/e2e_test.yml

```diff
@@ -26,7 +26,6 @@ jobs:
         with:
           build-ui: true
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          log-filename: e2e-${{ matrix.job }}.log
```

Comment:
@relrod This `log-filename` is an input to the other action (upload_awx_devel_logs), not this action. It seems to have not failed anything, but this seemed to be a bug.

I could, however, see the argument for having `run_awx_devel` call `upload_awx_devel_logs` in the same way that I'm now having it call the image build action. In that case, it would be an input, and we also would be more compact by never calling `upload_awx_devel_logs`

But that would require adding it as something like a "finalizer" step, which needs research.

Reply:
looks like it's not a thing https://github.com/actions/runner/issues/1478

Reply:
Yeah that is what I really wanted to do originally, but there's no way when using a composite action. :(